src/UI/Button.elm

```diff
@@ -505,12 +505,12 @@ toggleView cfg size hint toggleMsg current =
             iconLayout size
 
         attrs =
-            [ ARIA.roleAttr ARIA.roleButton
-            , Primitives.roundedBorders
+            [ Primitives.roundedBorders
             , paddings
             , borders
             , Events.onClick <| toggleMsg (not current)
             ]
+                ++ ARIA.toElementAttributes ARIA.roleButton
```

Comment:
Maybe it makes sense only to have a function called ARIA.roleButtonAttributes returning the list of Attributes? why do we need to have this type of function (`toElementAttributes`)?

Reply:
There are some ARIA attributes like "aria-describedby" that doesn't need to be attached to a role, the idea is that we may put those in the future using with-pattern.